airflow/contrib/kubernetes/kubernetes_request_factory/kubernetes_request_factory.py

```diff
@@ -187,12 +187,17 @@ def extract_image_pull_secrets(pod, req):
                 'name': pull_secret
             } for pull_secret in pod.image_pull_secrets.split(',')]
 
+    @staticmethod
+    def extract_security_context(pod, req):
+        if pod.security_context is None:
+            return
+
+        # Add security_context if not present in spec
+        req['spec']['securityContext'] = req['spec'].setdefault('securityContext', {})
```

Comment:
`setdefault` modifies in place, so we don't need to/shouldn't assign the result.

Reply:
Tiny nit: you've moved the location of this method in the file - could you put it back after `extract_tolerations` please?